app/builders/v2/report_builder.rb

```diff
@@ -121,4 +129,27 @@ def avg_first_response_time_summary
 
     avg_frt
   end
+
+  def agent_metrics
+    users = @account.users
+    users = users.where(id: params[:user_id]) if params[:user_id].present?
+    users.each_with_object([]) do |user, arr|
+      @user = user
+      arr << {
+        user: { id: user.id, name: user.name, thumbnail: user.avatar_url },
+        metric: conversations
+      }
+    end
+  end
+
+  def conversations
+    @open_conversations = scope.conversations.open
+    first_response_count = scope.reporting_events.where(name: 'first_response', conversation_id: @open_conversations.pluck('id')).count
```

Comment:
This creates an IN query, I'm not sure whether it is performant. @sojan-official

Reply:
discussed multiple options with @aswindevps  and amidst the ones which we came up to get this metric, this seemed the most efficient one.
- Even at 10000 open conversations for the agent, it still shouldn't be an immediate concern. 

ref: https://stackoverflow.com/a/47729784/939299

Reply:
Sounds good, can you please check the PR and see if it is good to merged?